utils_test.go

```
@@ -0,0 +1,55 @@
+	if err != nil {
+		t.Fatal(err)
+	}
+	req.Header.Set("Content-Type", w.FormDataContentType())
+	if err := req.ParseMultipartForm(1 << 20); err != nil {
+		t.Fatal(err)
+	}
+	file, header, err := req.FormFile("file")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer file.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `file.Close` is not checked

(errcheck)